commands/countlinetotal/index.js

```diff
@@ -35,14 +35,11 @@ module.exports = {
 			const originalSize = Number(history.Result.match(/([\d.]+) GB of space/)[1]);
 			const rate = sb.Utils.round((currentSize - originalSize) / days, 3);
 			const megabytesPerHour = sb.Utils.round(rate * 1024 / 24, 3);
-			const fillDate = new sb.Date().addDays((220 - currentSize) / rate); // 238 GB minus an estimate of ~18GB of other stuff
-			historyText = (megabytesPerHour === 0) ? 
-				(sb.Utils.tag.trim `
-					Lines are added at a rate of ~${megabytesPerHour} MB/hr.
-					At this rate, its impossible to calculate when Supibot's hard drive will fill.`) :
-				(sb.Utils.tag.trim `
-					Lines are added at a rate of ~${megabytesPerHour} MB/hr.
-					At this rate, Supibot's hard drive will run out of space approximately on ${fillDate.format("Y-m-d")}.`)
+			const fillDate = new sb.Date().addDays((875 - currentSize) / rate); // 931 GB minus an estimate of ~56GB of other stuff
+			history = `Lines are added at a rate of ~${megabytesPerHour} MB/hr. `
+			historyText += (megabytesPerHour === 0) ? 
+				(`At this rate, its impossible to calculate when Wanductbot's hard drive will fill.`) :
+				(`At this rate, Wanductbot's hard drive will run out of space approximately on ${fillDate.format("Y-m-d")}.`)
```

Comment:
Keep the condition in parens, remove parens from each template string, and move `?` and `: ` operators to starts of their respective lines. For inspiration, check other commands.